mod-dcb/src/main/resources/volaris/mod-dcb/features/cancelling-lending-flow.feature

```diff
@@ -0,0 +1,208 @@
+Feature: Testing Lending Flow Cancellation
+
+  Background:
+    * url baseUrl
+
+    * callonce login testAdmin
+    * def okapitokenUser = okapitoken
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json, text/plain'  }
+    * configure headers = headersUser
+    * callonce variables
+
+  Scenario: Cancel DCB Transaction manually
```

Comment:
Please add one more scenario that create a transaction, cancel the request by directly calling request API(Mocking UI action), then verify the status of the transaction. It should be cancelled.

Reply:
fixed